main/src/cgeo/geocaching/log/LogTemplateProvider.java

```diff
@@ -285,6 +286,17 @@ public String getValue(final LogContext context) {
                 return StringUtils.EMPTY;
             }
         });
+        templates.add(new LogTemplate("TYPE",  R.string.init_signature_template_type) {
+            @Override
+            public String getValue(final LogContext context) {
+                final Geocache cache = context.getCache();
+                if (cache != null) {
+                    final CacheType cacheType = cache.getType();
+                    return cacheType.pattern;
```

Comment:
`CacheType.pattern` is the non-translatable string used by GPX files etc. Should use `CacheType.stringId` instead AFAICS, which is the translated type identifier.

Reply:
Looks like the translated stringId is private, but comes through translated with getL10n() - not sure what the function is named for? .. But seems to be used in many places to get the translated name.

Reply:
@bmuzzin
Thanks for updating the PR.
BTW: "L10n" stands for "Localization" which starts with a "L", ends with a "n" and has 10 letters in between...